tests/ballerina-integration-test/src/test/resources/resiliency/resiliencyservices/http_failover_05.bal

```diff
@@ -209,67 +106,8 @@ service mock05 bind backendEP05 {
-            runtime:sleep(30000);
-        }
         http:Response response = new;
-        if (req.hasHeader(mime:CONTENT_TYPE)
-            && req.getHeader(mime:CONTENT_TYPE).hasPrefix(http:MULTIPART_AS_PRIMARY_TYPE)) {
-            match req.getBodyParts() {
-                // Setting the error response in case of an error
-                error err => {
-                    log:printError(err.message);
-                    response.setPayload("Error in decoding multiparts!");
-                    response.statusCode = 500;
-                }
-                // Iterate through the body parts.
-                mime:Entity[] bodyParts => {
-                    foreach bodyPart in bodyParts {
-                        if (bodyPart.hasHeader(mime:CONTENT_TYPE)
-                            && bodyPart.getHeader(mime:CONTENT_TYPE).hasPrefix(http:MULTIPART_AS_PRIMARY_TYPE)) {
-                            mime:Entity[] childParts = check bodyPart.getBodyParts();
-                            foreach childPart in childParts {
-                                // When performing passthrough scenarios, message needs to be built before
-                                // invoking the endpoint to create a message datasource.
-                                var childBlobContent = childPart.getByteArray();
-                            }
-                            io:println(bodyPart.getContentType());
-                            bodyPart.setBodyParts(untaint childParts, contentType = untaint bodyPart.getContentType());
-                        } else {
-                            var bodyPartBlobContent = bodyPart.getByteArray();
-                        }
-                    }
-                    response.setBodyParts(untaint bodyParts, contentType = untaint req.getContentType());
-                }
-            }
-        } else {
-            response.setPayload("Mock Resource is Invoked.");
-        }
-        caller->respond(response) but {
-            error e => log:printError(
-                           "Error sending response from mock service", err = e)
-        };
-
-    }
-}
-
-// Define the sample service to mock connection timeouts and service outages.
-@http:ServiceConfig {
-    basePath: "/statuscodes"
-}
-service failureStatusCodeService05 bind backendEP05 {
-    @http:ResourceConfig {
-        methods: ["POST", "PUT", "GET"],
-        path: "/"
-    }
-    errorStatusResource(endpoint caller, http:Request req) {
-        http:Response outResponse = new;
-        outResponse.statusCode = 503;
-        outResponse.setPayload("Failure status code scenario");
-        caller->respond(outResponse) but {
-            error e => log:printError(
-                           "Error sending response from mock service", err = e)
-        };
+        response.setPayload("Mock Resource is Invoked.");
+        caller->respond(response) but {error e => log:printError("Error sending response from mock service", err = e)};
```

Comment:
Shall we directly send the payload as response is not involved with any other operations??
caller->respond("Mock Resource is Invoked.") but {error e => log:printError("Error sending response from mock service", err = e)};